LogMonitor/docs/README.md

```diff
@@ -0,0 +1,188 @@
+# Log Monitor Documentation
+
+**Contents:**
+
+- [ETW Monitoring](#etw-monitoring)
+- [Event Log Monitoring](#event-log-monitoring)
+- [Log File Monitoring](#log-file-monitoring)
+- [Process Monitoring](#process-monitoring)
+
+## ETW Monitoring
+
+### Description
+
+ETW (Event Tracing for Windows) is a system for routing events. ETW is primarily intended for diagnostic purposes and is optimized to minimize impact on the overall system performance. Please note that ETW is different from the Windows Event Log.
+
+To monitor ETW logs using Log Monitor you require a valid _provider GUID_ (Globally Unique Identifier).
+
+The _provider GUID_ that you pass in the JSON config file provider must be registered with ETW. Log Monitor currently allows you to monitor events generated by MOF (classic) providers, manifest-based providers and Trace Logging providers. _**Note**: we do not monitor log events from WPP providers._
+
+You can see all the providers on your system using the `logman query providers` command.
+
+To confirm if you have a valid provider name or valid provider GUID you can type the command below in command prompt or PowerShell.
+
+```cmd
+logman query providers | findstr "<GUID or Provider Name>" 
+```
+
+### Configuration
+
+- `type` (required): This indicates the type of log you want to monitor for. It should be `ETW`. 
+- `eventFormatMultiLine` (optional): This is Boolean to indicate whether you want the logs displayed with or without new lines. It is initially to True ad you can set it to False depending on how you want to view the logs on the console.
```

Comment:
It is initially set to True ad you can set it to False depending on how you want to view the logs on the console. [Missing set to make it grammatically correct].